### ISSUE TITLE
css TRANSITION + POSITION is not available in ReFlow

### ISSUE THREAD

**Issue by ttodua** [edge-reflow/issues#175]
css TRANSITION property is not available..

also, although there is POSITION:RELATIVE, but  the properties for RELATIVE POSITION [ TOP,LEFT,RIGHT,BOTTOM]  is not available.. (for ABSOLUTE, YES, there is...)

**Comment by vmitnick** [comment]
That is correct, there are a number of CSS properties that Reflow does not currently support.

**Comment by ttodua** [comment]
..
